ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/QueryDescription.java

```diff
@@ -116,7 +116,13 @@ public String getExecutionPlan() {
   // kept for backwards compatibility
   @JsonProperty("state")
   public Optional<String> getState() {
-    return Optional.of(ksqlHostQueryStatus.toString());
+    if (ksqlHostQueryStatus.isEmpty()) {
+      return Optional.empty();
+    }
+
+    return Optional.of(
+        (ksqlHostQueryStatus.containsValue(KsqlQueryStatus.ERROR)
```

Comment:
Aren't there other states like `REBALANCING`, `PENDING_SHUTDOWN` etc.?  Should we document our rationale for clubbing all non-error states as `RUNNING`?

Reply:
I was basing this off of the upcoming query error handling work being done. In the design doc, we're only exposing `RUNNING` and `ERROR`. Only having these two states probably makes more sense once that work is done, but I preemptively added them to `KsqlQueryStatus`.